src/algebra/quadratic_discriminant.lean

```diff
@@ -0,0 +1,176 @@
+/-
+  Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
+  Released under Apache 2.0 license as described in the file LICENSE.
+  Authors: Zhouhang Zhou
+  -/
+
+import algebra.ordered_field
+import tactic.linarith tactic.ring
+
+/-!
```

Comment:
I think we should maintain the convention of keeping all the lines flush left in a long comment.